spec/tags/do_spec.cr

```diff
@@ -0,0 +1,37 @@
+require "../spec_helper.cr"
+
+struct Crinja::Value
+  def push(value)
```

Comment:
I'd rather not have this method in the public namespace, even if it's just in the specs.
It's also unnecessary: You can just call `array.as_a.push(item)` in the function implementation.